File: fjt.api/models/ReportChangeLogs.js
const Sequelize = require('sequelize');

module.exports = (sequelize) => {
    const ReportChangeLogs = sequelize.define('ReportChangeLogs', {
        id: {
            allowNull: false,
            autoIncrement: true,
            primaryKey: true,
            type: Sequelize.INTEGER
        },
        reportId: {
            type: Sequelize.INTEGER,
            allowNull: false
        },
        startActivityDate: {
            type: Sequelize.DATE,
            allowNull: false
        },
        endActivityDate: {
            type: Sequelize.DATE,
            allowNull: true
        },
        activityStartBy: {
            type: Sequelize.INTEGER,
            allowNull: false
        },
        isDeleted: {
            type: Sequelize.BOOLEAN,
            allowNull: true
        },
        createdBy: {
            type: Sequelize.STRING,
            allowNull: true
        },
        updatedBy: {
            type: Sequelize.STRING,
            allowNull: true
        },
        deletedBy: {
            type: Sequelize.STRING,
            allowNull: true
        },
        createByRoleId: {
            type: Sequelize.INTEGER,
            allowNull: true
        },
        updateByRoleId: {
            type: Sequelize.INTEGER,
            allowNull: true
        },
        deleteByRoleId: {
            type: Sequelize.INTEGER,
            allowNull: true
        }
    }, {
        tableName: 'report_change_logs',
        paranoid: true
    });

    ReportChangeLogs.associate = (models) => {
        ReportChangeLogs.belongsTo(models.ReportMaster, {
            foreignKey: 'reportId',
            as: 'ReportMaster'
        });
        ReportChangeLogs.belongsTo(models.User, {
            as: 'user',
            foreignKey: 'activityStartBy'
        });
    };

    return ReportChangeLogs;
};

File: FJT.ReportDesigner/MySqlDBModel/report_change_logs.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace FJT.ReportDesigner.MySqlDBModel
{
    public class report_change_logs
    {
        [Key]
        public int id { get; set; }

        public int reportId { get; set; }

        public DateTime startActivityDate { get; set; }

        public DateTime? endActivityDate { get; set; }

        public int activityStartBy { get; set; }

        public bool isDeleted { get; set; }

        [StringLength(255)]
        public string createdBy { get; set; }

        [StringLength(255)]
        public string updatedBy { get; set; }

        [StringLength(255)]
        public string deletedBy { get; set; }

        public DateTime createdAt { get; set; }

        public DateTime? updatedAt { get; set; }

        public DateTime? deletedAt { get; set; }

        public int? createByRoleId { get; set; }

        public int? updateByRoleId { get; set; }

        public int? deleteByRoleId { get; set; }

        [ForeignKey("reportId")]
        public virtual reportmaster reportmaster { get; set; }
    }
}
